### PR TITLE
[WEBSITE] Old career listing frontend delete

### PR DESCRIPTION
**Pull request by FaithDaka**
#### Summary of Changes (What does this PR do?)

- Checks a career listing's closing date to show only newer careers.
- Careers still kept in backend
- Portal update

#### Status of maturity (all need to be checked before merging):

- [x] I've tested this locally
- [x] I consider this code done
- [x] This change ready to hit production in its current state
- [x] The title of the PR states what changed and the related issues number (used for the release note).

#### How should this be manually tested?

- Using the deploy preview add a career through admin portal, with a past closing date.
- Check if it shows on the frontend.
- Do the same for a future date.
- Check if it shows on the frontend.

### CODE REVIEW

File: website/frontend/src/pages/CareerPage.js [hunk 1]
```diff
@@ -9,151 +9,161 @@ import { isEmpty } from 'underscore';
...
-                        </div>
-                    </div>
+                  <span></span>
                 )}
+              </div>
+              {groupedKeys.length > 0 ? (
+                groupedKeys.map((groupedKey) => {
+                  const departmentListing = groupedListing[groupedKey];
+                  const listing = departmentListing.filter(
+                    (job) => new Date(job.closing_date).getTime() >= new Date().getTime()
```
Comment (FaithDaka) [inline website/frontend/src/pages/CareerPage.js:138]: Filter job listings by newer dates in order to show on the frontend

File: website/backend/career/admin.py [hunk 2]
```diff
@@ -33,11 +33,12 @@ class BulletDescriptionInline(nested_admin.NestedTabularInline):
...
 class CareerAdmin(nested_admin.NestedModelAdmin):
-    list_display = ('id', 'title', 'closing_date')
+    list_display = ('title','department', 'closing_date', 'author')
     readonly_fields = ('id', 'author', 'created', 'updated_by', 'modified')
 
     fields = ('id', "title", "department", "type", "apply_url", "closing_date", 'author', 'created', 'updated_by', 'modified')
     list_per_page = 10
-    search_fields = ('title',)
+    search_fields = ('title','department')
+    list_filter=('department','closing_date')
```
Comment (FaithDaka) [inline website/backend/career/admin.py:42]: Added filtering and search options for the portal

File: website/frontend/src/pages/CareerPage.js (hunk 1 above)
Comment (Baalmart) [reply website/frontend/src/pages/CareerPage.js:138]: Great!

**Review by Baalmart — APPROVED** [review]
This LGTM, thanks @FaithDaka

**Pull request merged**